internal/core/src/indexbuilder/VecIndexCreator.cpp

```diff
@@ -75,14 +75,6 @@ VecIndexCreator::parse_impl(const std::string& serialized_params_str, knowhere::
     /************************** PQ Params *****************************/
     check_parameter<int>(conf, knowhere::indexparam::PQ_M, stoi_closure, std::nullopt);
```

Comment:
Emm.. just some comment. Why we put all params for all indexes here......

Reply:
I don't know why.
Actually, knowhere already has parameter check, segcore should call knowhere's parameter check API.
This part of code is redundant.